site/pages/about-us/principles-slice/principle/style.css

```diff
@@ -0,0 +1,41 @@
+@value badgerWhite, badgerRed, badgerBlack from "../../../../css/_colors.css";
+@value mediumScreen, largeScreen, maxWidth from "../../../../css/_sizes.css";
+
+.container{
+	display: flex; 
+	padding-top: 20px;
+}
+
+.leftCol{
+}
+.rightCol{
+	padding-left: 30px;
+	max-width: 670px;
+}
+.number {
+	padding-top: 5px;
+  composes: boldSansSerif from "../../../../css/typography/_fonts.css";
```

Comment:
Yeah - I don't actually see that in my IDE...suspect it's to do with tab vs space indenting.